evap/evaluation/models.py

```diff
@@ -362,12 +370,12 @@ def textanswer_set(self):
     @property
     def open_textanswer_set(self):
         """Pseudo relationship to all text answers for this course"""
-        return TextAnswer.objects.filter(contribution__in=self.contributions.all(), checked=False)
+        return TextAnswer.objects.filter(contribution__in=self.contributions.all(), state='')
```

Comment:
suppose joe looks at this code and says "here a state is checked for empty string, i guess that should be some real state?" "no, emtpystring is also a state" "alright what does it mean?" "well, it means that the comment has not been reviewed yet"

if you change this to use constants, this is a bit moot, but i think an emptystring as a state is not acceptable.